### PR TITLE
Improve discord webhooks

### PR DESCRIPTION
**Pull request by mdiehr**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -126,8 +126,8 @@ try {
 		twitter.postTweet(headline.text);
 	}
 	// Notify about tweet
-	Discord.sendEmbed(`${headline.text}`);
+	Discord.sendEmbed(`${headline.text}`, `https://twitter.com/MTGnewsbot`);
```
Comment (jwoolley) [inline bot.js]: this is a great config parameter candidate

File: run_bot.sh [hunk 2]
```diff
@@ -21,6 +21,9 @@ git pull
 # install dependencies
 npm install --production
 
+# install fonts
+cp ./src/data/fonts/*.ttf /usr/local/share/fonts
```
Comment (jwoolley) [inline run_bot.sh]: this too probably

File: bot.js (hunk 1 above)
Comment (mdiehr) [inline bot.js]: yeah, def

File: run_bot.sh (hunk 2 above)
Comment (jwoolley) [inline run_bot.sh]: or maybe not, since this is a shell script, lol

Comment (mdiehr) [inline run_bot.sh]: not sure about the best way to configure the shell scripts. any ideas?

**Pull request merged**